browser/react/components/App.js

```diff
@@ -1,16 +1,24 @@
 import React from 'react';
-
 import '../../aframeComponents/scene-load';
+import '../../aframeComponents/aframe-hyperlink';
 import '../../aframeComponents/aframe-minecraft';
 
 export default function App (props) {
   return (
     <div style={{ width: '100%', height: '100%' }}>
       <a-scene id="scene" scene-load>
-        <a-entity geometry="primitive: sphere; radius: 1.25" position="0 1.25 -1" material="color: #EF2D5E"></a-entity>
-        <a-entity geometry="primitive: box; width: 1; height: 1; depth: 1" position="-1 0.5 1" material="color: #4CC3D9"></a-entity>
-        <a-entity geometry="primitive: cylinder; radius: 0.5; height: 1.5" position="1 0.75 1" material="color: #FFC65D"></a-entity>
-        <a-entity geometry="primitive: plane; width:4; height: 4" rotation="-90 0 0" material="color: #7BC8A4"></a-entity>
+        <a-assets>
```

Comment:
Whoops. Forgot I was in markdown-land and my tags need to be escaped. `<a-assets>` must be a direct child of `<a-scene>`. To move between pages without bombing out of VR Mode on the Daydream, I had to make the parent div of the React components an `<a-entity>`. The `<a-assets>` must therefore be centralized within the parent `App` component to be directly beneath the `<a-scene>`. We could theoretically stop using the A-Frame asset management system and load the resources directly via something like url('/sean/tape-reader.dae'), but I suspect that would be less performant.